### PR TITLE
Added Volume.js into the Math file

### PR DESCRIPTION
**Pull request by leeyan44**
# Welcome to JavaScript community

[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### **Describe your change:**

* [v] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [v] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
* [v] This pull request is all my own work -- I have not plagiarized.
* [v] I know that pull requests will not be merged if they fail the automated tests.
* [v] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [v] All new JavaScript files are placed inside an existing directory.
* [v] All filenames should use the UpperCamelCase (PascalCase) style.  There should be no spaces in filenames.
     **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
* [v] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [v] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Maths/Volume.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+*/
+const volHemisphere = (radius) => {
+  isNumber(radius, 'Radius')
+  return (2.0 * Math.PI * radius ** 3)/ 3.0
+}
+
+const isNumber = (number, noName = 'number') => {
+  if (typeof number !== 'number') {
+    throw new TypeError('The ' + noName + ' should be Number type')
+  } else if (number < 0) {
```
Comment (raklaptudirm) [inline Maths/Volume.js:118]:
```suggestion
  } else if (number < 0 || (!Number.isFinite(number))) {
```
`Number.isFinite()` prevents someone from entering `NaN` or `Infinity` as a number, which would previously pass your tests.

**Review by raklaptudirm — CHANGES_REQUESTED** [review]
Create a `Maths/test/Volume.test.js` file for testing the algorithm. We use `jest` for testing. If you want to know how to write tests, refer to the other files in the `test` folder.

**Comment by raklaptudirm** [comment]
@leeyan44 Why did you close the PR?

**Comment by leeyan44** [comment]
Sorry, I’m new to this. What should i do now?

**Comment by raklaptudirm** [comment]
@leeyan44 Thats ok. Now you see our repositories have a style guide to keep code consistent and readable. It seems like your code does not follow the style. You need to format your code with [`standard.js`](https://standardjs.com/). Download the cli-app and run the command `standard --fix .` on the repository. If you need any help, ask me.

**Review by raklaptudirm — APPROVED** [review]

**Pull request merged**